recipes/google-cloud-cpp/2.x/test_package/conanfile.py

```diff
@@ -14,6 +14,14 @@ class TestPackageConan(ConanFile):
     def requirements(self):
         self.requires(self.tested_reference_str)
 
+    def build_requirements(self):
+        if not self._is_legacy_one_profile:
+            self.tool_requires("grpc/<host_version>")
```

Comment:
My only concern now is that the test package also needs the dependency, not opposing it, but having a bit more insight into why this is the case would be great!

Reply:
Good catch. Removed.

I was copying gRPC who run protobuf in their test package:

https://github.com/SpaceIm/conan-center-index/blob/9991d8f1f7c29bd7e7ee8ea40bf4cac593fc8147/recipes/grpc/all/test_package/CMakeLists.txt#L21-L30

But I don't think `google-cloud-cpp` needs anything like that.